### ISSUE TITLE
bogus syntax error

### ISSUE THREAD

**Issue by glensc** [JakubOnderka/PHP-Parallel-Lint#88]
i do not know what is special with this file that phplint fails. what is more weirder that the error is not triggered with `-s` flag even the file does not use short tags.

```
➔ ./parallel-lint --help
PHP Parallel Lint version 0.9.2
-------------------------------
Usage:
    parallel-lint [sa] [-p php] [-e ext] [-j num] [--exclude dir] [files or directories]

Options:
    -p <php>        Specify PHP-CGI executable to run (default: 'php').
    -s, --short     Set short_open_tag to On (default: Off).

➔ ./parallel-lint preview.phtml
PHP 5.6.27 | 10 parallel jobs
X                                                            1/1 (100 %)


Checked 1 files in 0.6 seconds
Syntax error found in 1 file

------------------------------------------------------------
Parse error: preview.phtml:13
    11|         </div>
    12|         <?php $i++; ?>
  > 13| <?php endforeach; ?>
    14| </div>
Unexpected 'endforeach' (T_ENDFOREACH)

➔ ./parallel-lint preview.phtml -s
PHP 5.6.27 | 10 parallel jobs
.                                                            1/1 (100 %)


Checked 1 files in 0.6 seconds
No syntax error found

➔

```

the input file in question:

```html
<div class="customblock-preview-galleries customblock-preview-layout-<?= $this->layout ?>">
<?php $i = 0; ?>
<?php foreach($this->galleries as $gallery): ?>
        <div class="customblock-preview-gallery customblock-preview-gallery-<?= $i + 1 ?>">
                <div class="customblock-preview-gallery-cover">
                        <?php if(!empty($gallery->extra->cover)): ?>
                        <img src="<?= $this->imageUrl($gallery->extra->cover->assetId, 520, 250) ?>">
                        <? endif; ?>
                </div>
                <div class="customblock-preview-gallery-title"><?= $gallery->title ?></div>
        </div>
        <?php $i++; ?>
<?php endforeach; ?>
</div>
```

**Comment by glensc** [comment]
```
➔ php -d error_reporting=E_ALL -n -l preview.phtml
No syntax errors detected in preview.phtml

➔ php -d short_open_tag=Off -d error_reporting=E_ALL -n -l preview.phtml

Parse error: syntax error, unexpected 'endforeach' (T_ENDFOREACH) in preview.phtml on line 13
Errors parsing preview.phtml

➔ php --version
PHP 5.6.27 (cli) (built: Nov 21 2016 23:08:15)
```

**Comment by glensc** [comment]
god damn it. found the short tag. closing as report is bogus

```diff
[~/scm/php/PHP-Parallel-Lint (master)★] ➔ d preview.phtml
--- PHP-Parallel-Lint/preview.phtml~    2016-12-08 15:21:22.789994022 +0200
+++ PHP-Parallel-Lint/preview.phtml    2016-12-08 15:21:24.390004151 +0200
@@ -5,7 +5,7 @@
         <div class="customblock-preview-gallery-cover">
             <?php if(!empty($gallery->extra->cover)): ?>
             <img src="<?= $this->imageUrl($gallery->extra->cover->assetId, 520, 250) ?>">
-            <? endif; ?>
+            <?php endif; ?>
         </div>
         <div class="customblock-preview-gallery-title"><?= $gallery->title ?></div>
     </div>
(END)
```
